src/pages/Content/index.js

```diff
@@ -80,8 +80,18 @@ function listener(e) {
   }
 }
 
+// The function waits between play and clicks
+async function wait(delay) {
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      resolve();
+    }, delay);
+  });
+}
+
 // The function plays a recording when needed
 async function playRecording(recordedEvents) {
+  await wait(10000); // Wait for 3 seconds
```

Comment:
Why are you doing this?

Reply:
Should be wait(3000). 
In Trello todo list, there is a task named "wait between play and clicks".
Am I misunderstood?